### ISSUE TITLE
using the pretained model and finetuning,but get a low accuracy with lfw，how to explain...

### ISSUE THREAD

**Issue by qianchen94** [davidsandberg/facenet#774]
i used the model trained by the author about 20180402-114759 using VGGface2 to finetune.
however, i use the MS-Cleb-1M as training data，but accuracy is about 0.70 after epcoch 200. 
i wonder is there something wrong with my parameters or something else?
tks a lot~
dose any can help me ?
max_nrof_epochs: 300
lfw_pairs: ./data/pairs.txt
embedding_size: 128
learning_rate_decay_factor: 1.0
images_per_person: 40
models_base_dir: ./models/facenet
lfw_nrof_folds: 10
seed: 666
pretrained_model: ./models/facenet/20180402-114759.ckpt-275
weight_decay: 0.0001
learning_rate_decay_epochs: 100
data_dir: ./datasets/MS-Cele-1M
people_per_batch: 45
epoch_size: 1000
learning_rate_schedule_file: ./data/learning_rate_schedule_classifier_meceleb.txt
learning_rate: -1.0
gpu_memory_fraction: 1.0
optimizer: ADAGRAD
model_def: models.inception_resnet_v1
random_flip: False
batch_size: 90
image_size: 160
alpha: 0.2
keep_probability: 0.8
logs_base_dir: ./logs
moving_average_decay: 0.9999
lfw_dir: ./datasets/lfw/lfw_mtcnnpy_160
random_crop: False
